### ISSUE TITLE
What's new pages should be 

### ISSUE THREAD

**Issue by tnir** [rubygems/bundler-site#827]
![bundler io_v1 12_whats_new html](https://user-images.githubusercontent.com/10229505/182031308-6f372dfd-ce27-43f5-b301-8f72fe81b50f.png)

`What's New in each Release` does not provide any information to readers at all.
